Remotion/Web/ClientScript/Scripts/ListMenu.ts

```diff
@@ -149,65 +159,71 @@ class ListMenu
 
       if (itemInfo.DiagnosticMetadata)
       {
+        // TODO: merge loops?
```

Comment:
We can merge the loops on both occasions, JS is single threaded so the behavior is not changed at all.

Just make sure to but the seconds loop content into the `else` since the first loop removes elements that would not be iterated in the second loop.

Reply:
Is the delete even needed here? I do not really see the purpose, imo the condition could be inverted and the delete removed.

Reply:
Yes since we no longer need to pass the object to a jQuery method we also do not need to "clean" the object of any empty values.